### PR TITLE
feat(Spanner): add samples for jsonb

### PR DESCRIPTION
**Pull request by NivedhaSenthil**
## Description

Fixes #<ISSUE-NUMBER>

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [ ] **Tests** pass
- [ ] **Lint** pass: `bundle exec rubocop`
- [ ] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: spanner/spanner_postgresql_jsonb_query_parameter.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  sql_query = <<~QUERY
+    SELECT venueid, venuedetails 
+    FROM Venues
+    WHERE CAST(venuedetails ->> 'rating' AS INTEGER) > $1
+  QUERY
+
+  results = client.execute sql_query, params: { p1: 5 }
```
Comment (bajajneha27) [inline spanner/spanner_postgresql_jsonb_query_parameter.rb:29]: What does the `param` mean here? 
I think it'll be good if we add some comments here.

File: spanner/spec/spanner_postgresql_jsonb_add_column.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+require_relative "../spanner_postgresql_jsonb_add_column"
+
+describe "Google Cloud Spanner Postgres examples" do
+  before :each do
+    cleanup_database_resources
+  end
+
+  after :each do
+    cleanup_database_resources
+    cleanup_instance_resources
```
Comment (bajajneha27) [inline spanner/spec/spanner_postgresql_jsonb_add_column.rb:26]: Just curious, where do we create instance resources?

**Review by bajajneha27 — COMMENTED** [review]
Overall LGTM. Have some minor comments/questions

File: spanner/spanner_postgresql_jsonb_query_parameter.rb (hunk 1 above)
Comment (NivedhaSenthil) [reply spanner/spanner_postgresql_jsonb_query_parameter.rb:29]: Param here is the key value pair for parameterised query's parameter.
Sure will add a comment here 👍

File: spanner/spec/spanner_postgresql_jsonb_add_column.rb (hunk 2 above)
Comment (NivedhaSenthil) [reply spanner/spec/spanner_postgresql_jsonb_add_column.rb:26]: Instance is not created in the spec.. it is passed on via env variable `GOOGLE_CLOUD_SPANNER_TEST_INSTANCE `

Comment (NivedhaSenthil) [reply spanner/spec/spanner_postgresql_jsonb_add_column.rb:26]: Just realised we don't have to cleanup instance resource here as we don't create any for this particular test. 
Thanks for pointing out dint realise it earlier 👍

**Review by bajajneha27 — APPROVED** [review]

**Pull request merged**